docs/source/getting-started/tabular-data/data-prep.ipynb

```diff
@@ -0,0 +1,279 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Preparation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section explains the limitations of the `with_columns` and `filter` methods in our `dp` plugin for Polars. \n",
+    "\n",
+    "Although these methods function similarly to normal Polars versions, in this context there are privacy implications that may not be immediately apparent. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import polars as pl \n",
+    "import opendp.prelude as dp\n",
+    "import seaborn as sns \n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "dp.enable_features(\"contrib\")\n",
+    "sns.set_theme(style='darkgrid')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
```

Comment:
Again (same comment as in the other files), maybe put the output in a box and explicitly state that. Just to make it neater.

Reply:
yeah, I can try to disable the work but this isn't supposed to be an output, it's just reading in the file